lsp/src/dune

```diff
@@ -12,6 +13,20 @@
 (ocamllex
  (modules text_document_text))
 
+(library
+ (name ocaml_uri)
+ (public_name lsp._ocaml_uri)
```

Comment:
I might be missing something, but why do we need this wrapper library?

Reply:
I'm probably missing something too, but if I use `Uri` inside `uri.ml` by simply adding `uri` to the lsp libraries, I get this:

```
File "lsp/src/uri.ml", line 3, characters 9-14:
3 | type t = Uri.t
             ^^^^^
Error: The module Uri is an alias for module Lsp__Uri, which is missing
```

Reply:
I see. That's quite annoying. In this case, I think we should just rename our `uri.ml` to something like `uri0.ml` then. And then rename it back in `lsp.ml` and `lsp.mli`

Reply:
Ah, good idea, it's a much better solution

Reply:
But there's no `lsp.ml` and `lsp.mli` 😅 
I'm going to create lsp.ml